subsys/pm/Kconfig

```diff
@@ -103,6 +103,13 @@ config PM_DEVICE_RUNTIME
 	  enabled, devices can be suspended or resumed based on the device
 	  usage even while the CPU or system is running.
 
+config PM_DEVICE_RUNTIME_EXCLUSIVE
+	depends on PM_DEVICE_RUNTIME
```

Comment:
nit: it should probably be `default y`

Reply:
I think we should be cautious here and not setting this to `y` to not disrupt the current behavior.

Reply:
Note that device PM coupled with system PM has issues now (to the point it's not much usable in practice), so I think it makes sense defaulting it to `y` when users enable runtime PM